integration_tests/samples/openid_connect/flask_example.py

```diff
@@ -0,0 +1,118 @@
+import json
+import logging
+import os
+import jwt
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
+
+client_id = os.environ["SLACK_CLIENT_ID"]
+client_secret = os.environ["SLACK_CLIENT_SECRET"]
+redirect_uri = os.environ["SLACK_REDIRECT_URI"]
+scopes = ["openid", "email", "profile"]
+
+from slack_sdk.web import WebClient
+from slack_sdk.oauth import OpenIDConnectAuthorizeUrlGenerator, RedirectUriPageRenderer
+from slack_sdk.oauth.state_store import FileOAuthStateStore
+
+state_store = FileOAuthStateStore(expiration_seconds=300)
+
+authorization_url_generator = OpenIDConnectAuthorizeUrlGenerator(
+    client_id=client_id,
+    scopes=scopes,
+    redirect_uri=redirect_uri,
+)
+redirect_page_renderer = RedirectUriPageRenderer(
+    install_path="/slack/install",
+    redirect_uri_path="/slack/oauth_redirect",
+)
+
+
+from flask import Flask, request, make_response
+
+app = Flask(__name__)
+app.debug = True
+
+
+@app.route("/slack/install", methods=["GET"])
+def oauth_start():
+    state = state_store.issue()
```

Comment:
What is this line doing?

Reply:
The line generates a new unique state parameter value. The state_store (`FileOAuthStateStore` in this example) stores the generated value in the local file system. And then, `state_store.consume(state)` in the redirect URL handler checks and deletes it. 

This SDK provides local files, Amazon S3, and RDB as the backends of `OAuthStateStore`: https://github.com/slackapi/python-slack-sdk/tree/main/slack_sdk/oauth/state_store 

If you switch to other implementation, this code works with the chosen backend without any changes.